cleartk-berkeleyparser/src/test/java/org/cleartk/berkeleyparser/DefaultBerkeleyTokenizerTest.java

```diff
@@ -0,0 +1,169 @@
+/* 
+ * Copyright (c) 2012, Regents of the University of Colorado 
+ * All rights reserved.
+ * 
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * For a complete copy of the license please see the file LICENSE distributed 
+ * with the cleartk-syntax-berkeley project or visit 
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.html.
+ */
+package org.cleartk.berkeleyparser;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import org.apache.uima.UIMAException;
+import org.apache.uima.analysis_engine.AnalysisEngineDescription;
+import org.apache.uima.fit.pipeline.SimplePipeline;
+import org.apache.uima.fit.util.JCasUtil;
+import org.cleartk.token.type.Sentence;
+import org.cleartk.token.type.Token;
+import org.junit.Test;
+
+/**
+ * <br>
+ * Copyright (c) 2010, Regents of the University of Colorado <br>
+ * All rights reserved.
+ * 
+ * @author Majid Laali
+ */
+public class DefaultBerkeleyTokenizerTest extends BerkeleyTestBase {
+
+  @Test
+  public void givenASentenctWhenTokenizingThenAllTokenAreReturned() throws UIMAException{
```

Comment:
Typo in test name.  Change `givenASentenctWhenTokenizingThenAllTokenAreReturned` to `givenASentenceWhenTokenizingThenAllTokenAreReturned`.